### DISCUSSION TITLE
How can I change Git Branch

### DISCUSSION THREAD

**Discussion by AbidHasanPiash** [STRINGLABITSOLUTIONS/Bhojon#179] [Q&A] [answered]
Git Branch changing script

**Comment by montasim** [comment, marked as answer]
git checkout branchName

**Comment by AbidHasanPiash** [comment]
Thanks

**Comment by Preetiraj3697** [comment]
git fetch origin
git branch 'give you branch name'
git checkout branch
git add.
git commit -m"add branch"
git push origin main/master
